From: Alberto Garcia <agarcia@igalia.com>
Subject: Add support for libaudiofile 0.3.x
Bug: http://sourceforge.net/tracker/?func=detail&aid=3508658&group_id=91293&atid=596650
Index: libspectrum/configure.in
===================================================================
--- libspectrum.orig/configure.in
+++ libspectrum/configure.in
@@ -178,10 +178,17 @@ if test "$withval" = no; then libaudiofi
 libaudiofile=yes)
 AC_MSG_RESULT($libaudiofile)
 if test "$libaudiofile" = yes; then
-  AM_PATH_AUDIOFILE(
-    0.2.3,
-    AC_DEFINE([HAVE_LIB_AUDIOFILE], 1, [Defined if we've got audiofile])
+  PKG_CHECK_MODULES(
+    AUDIOFILE,
+    audiofile,
+    AC_DEFINE([HAVE_LIB_AUDIOFILE], 1, [Defined if we've got audiofile]), []
   )
+  if test -z "$AUDIOFILE_LIBS"; then
+    AM_PATH_AUDIOFILE(
+      0.2.3,
+      AC_DEFINE([HAVE_LIB_AUDIOFILE], 1, [Defined if we've got audiofile])
+    )
+  fi
 fi
 
 AC_OUTPUT(
